src/main/msp/msp_box.c

```diff
@@ -205,6 +205,13 @@ void initActiveBoxIds(void)
         BME(BOXHORIZON);
         BME(BOXHEADFREE);
         BME(BOXHEADADJ);
+        BME(BOXFPVANGLEMIX);
+        if (featureIsEnabled(FEATURE_INFLIGHT_ACC_CAL)) {
+            BME(BOXCALIB);
+        }
+#if defined(USE_ACRO_TRAINER) && defined(USE_ACC)
+        BME(BOXACROTRAINER);
```

Comment:
```suggestion
    if (sensors(SENSOR_ACC)) {
        BME(BOXACROTRAINER);
    }
```

Reply:
isn't it already under `if (sensors(SENSOR_ACC))` statement in line 203?

Reply:
Yes was hidden from view. Ignore.